martijnversluis/ChordSheetJS
# Ensure full support for bridge, grid and tab directives

- Test all implemented aliases

For bridge, grid and tab sections:

- Ensure short aliases are in place
- Ensure paragraph type is parsed and rendered
- `isBridge()`, `isGrid()` and `isTab()` methods are added for `Line
